guides/source/active_storage_overview.md

```diff
@@ -961,10 +961,33 @@ class Uploader {
 
 NOTE: Using [Direct Uploads](#direct-uploads) can sometimes result in a file that uploads, but never attaches to a record. Consider [purging unattached uploads](#purging-unattached-uploads).
 
-Discarding Files Stored During System Tests
+Testing
 -------------------------------------------
 
-System tests clean up test data by rolling back a transaction. Because destroy
+Use [`fixture_file_upload`][] to test uploading a file in an integration or controller test.
+Rails handles it like any other param.
+
+```ruby
+class SignupController < ActionDispatch::IntegrationTest
+  test "signup" do
+    post signup_path, params: {
+      name: "Spongebob",
```

Comment:
The testing guide uses names of Rails team members and contributors when referencing users. Please use these instead for consistency and to avoid any potential copyright issues.

Reply:
Will do. I was copying the pattern here: https://github.com/rails/rails/blob/f95c0b7e96eb36bc3efc0c5beffbb9e84ea664e4/actionpack/lib/action_dispatch/testing/test_process.rb#L11 - should I change that too? (I realise that's not in a guide, but it is in public docs)

Reply:
Ah! Interesting. I'm a little more worried about the guides because of SEO. I don't see any policy on this sort of thing in the contributor's guide, but I would accept a change for this line as well.

Reply:
Updated both to be safe and consistent. https://github.com/rails/rails/pull/41543/files#diff-2f684945f008de83109c95e5234ed9c2c1cfc7abbfd55ec148b9a25d4c18c85dR11